src/components/stack/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment } from "react";
```

Comment:
```suggestion
import React from "react";
```

fragment is unused here. If you were using <Fragment key="123">, then you'd need it. but, as the code is using `<>`, it's not needed https://reactjs.org/docs/fragments.html#short-syntax 

Fixes the lgtm alert as well https://github.com/WPMedia/arc-themes-components/pull/25#issuecomment-1050247966

Reply:
From a previous coding attempt and it was just left it.  Has been removed